src/main/java/seedu/easylog/parser/ItemsParser.java

```diff
@@ -52,4 +66,15 @@ public static void processItemsInput(String itemsInput, ItemManager itemManager)
             ui.showItemsHelp();
         }
     }
+
+    /**
+     * Prompts the user for the item price.
+     *
+     * @return item price
+     */
+    public String processItemsPrice() {
+        String price = Constants.SCANNER.nextLine();
+
+        return price;
+    }
```

Comment:
Do together with item stock? If not for now since quite trivial can put inside the command as well